calico/calcollections.py

```diff
@@ -75,3 +75,89 @@ def resulting_size(self):
     @property
     def empty(self):
         return not (self.added_entries or self.removed_entries)
+
+
+class OneToManyIndex(object):
```

Comment:
I think strictly speaking it's more commonly called a `MultiDict`, at least in HTTP-land.